static_file_handler.cc

```diff
@@ -56,16 +62,30 @@ StaticFileHandler::Status StaticFileHandler::HandleRequest(const Request& reques
     if (ReadFile(file_path, file_content)) {
         response->SetStatus(Response::ResponseCode::OK);
 
-        std::string content_type = GetMimeType(file_path);
+        bool isMarkdown = false;
+        std::string content_type = GetMimeType(file_path, isMarkdown);
```

Comment:
do you think it would be better to have separate helper function that checks markdown? like
```c++
if (IsMarkdown()) {
  file_content = ProcessMarkdown(file_content);
}
```
so then we don't need to pass a bool into GetMimeType? I'm not sure if this is really a better solution, so feel free to go w what you think is better hahaha